src/info/langs/language.tera

```diff
@@ -2,142 +2,77 @@ use owo_colors::{
     AnsiColors,
     DynColors::{self, Ansi, Rgb},
 };
-use std::fmt;
-use std::fmt::Write;
-use strum::EnumIter;
 
 pub struct Colors {
     basic_colors: Vec<DynColors>,
     true_colors: Option<Vec<DynColors>>,
 }
 
-#[derive(Clone, PartialEq, Eq, Debug, clap::ValueEnum)]
-pub enum LanguageType {
-    Programming,
-    Markup,
-    Prose,
-    Data,
-}
-
-#[derive(Clone, Copy, PartialEq, Eq, Hash, EnumIter, clap::ValueEnum, Debug, Serialize)]
-#[allow(clippy::upper_case_acronyms)]
-#[clap(rename_all = "lowercase")]
-pub enum Language {
-    {% for language, attrs in languages -%}
-        {% if attrs.serialization %}#[clap(name="{{ attrs.serialization }}")]{% endif -%}
-        {{ language }},
-    {% endfor %}
-}
-
-impl fmt::Display for Language {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        match self {
-            {% for language, _ in languages -%}
-                Self::{{ language }} => write!(f, "{}", tokei::LanguageType::{{ language }}.name()),
-            {% endfor %}
-        }
-    }
-}
-
-impl From<tokei::LanguageType> for Language {
-    fn from(language: tokei::LanguageType) -> Self {
-        match language {
-            {% for language, _ in languages -%}
-                tokei::LanguageType::{{ language }} => Self::{{ language }},
-            {% endfor %}
-            _ => unimplemented!("Language {:?}", language),
-        }
+pub(crate) fn is_supported(language: &str) -> bool {
+    match language {
+        {% for language, attrs in languages -%}
+            r#"{{ language }}"# => true,
+        {% endfor %}
+        _ => false,
     }
 }
 
-impl From<Language> for tokei::LanguageType {
-    fn from(language: Language) -> Self {
-        match language {
-            {% for language, _ in languages -%}
-                Language::{{ language }} => tokei::LanguageType::{{ language }},
-            {% endfor %}
-        }
+pub(crate) fn get_ascii_art(s: &str) -> &'static str {
+    match s {
```

Comment:
won't that make it case sensitive, for example if I do `onefetch -a rust`, will it work ?

Reply:
Good point, I'll normalize the strings to lowercase for this one.